project/profiles/capacity4more/behat/features/group_content_access.feature

```diff
@@ -35,3 +35,21 @@ Feature: Group content access
     When  I am logged in as user "isaacnewton"
     Then  I visit "Test content" node of type "discussion"
     And   I should see "Access denied"
+
+  @javascript
+  Scenario: An anonymous user shouldn't be able to see a restricted group's content.
+    Given I am logged in as user "turing"
+    And   I change access of group "My test group" to "Restricted" with the domain "turingmachine.com"
+    When  I am an anonymous user
+    Then  I visit "Test content" node of type "discussion"
+    And   I should not see "Test content"
+    And   I should see "Please log in to continue"
+
+  @javascript
+  Scenario: A member of the organization should be able to see the group's content.
+    Given I am logged in as user "turing"
+    And   I change access of group "My test group" to "Restricted" with the domain "turingmachine.com"
```

Comment:
@ybaras 
You already invoked the same command at the previous scenario, hence the access expected to be like that already, so try to remove this line.